radix-engine-macros/src/decimal.rs

```diff
@@ -0,0 +1,64 @@
+use proc_macro::TokenStream;
+use quote::quote;
+use syn::{parse_macro_input, Expr};
+
```

Comment:
You can return a `Result`, which will be presented to users as a compile-time error.

Reply:
Sure, good point!